src/express/types.ts

```diff
@@ -49,3 +54,7 @@ export declare type PayloadRequest<U = any> = Request & {
     [slug: string]: (q: unknown) => Document;
   };
 };
+
+export interface PayloadRequestContext {
```

Comment:
I'd also drop the `Payload` off here too to make it `RequestContext` since it is already coming from Payload, the prefix is redundant.